GenderPayGap.WebUI/wwwroot/styles/components/_gpg-search-box.scss

```diff
@@ -11,7 +11,7 @@
         padding: 0;
         width: 40px;
         height: 40px;
-        background-image: url('/assets/images/search-button-white-and-black.png');
+        background-image: url('../../assets/images/search-button-white-and-black.png');

```

Comment:
Is this always correct?
I assume this `../../` makes it into the output CSS?
If so, this might break if we use it on a page at a different level of nesting

Reply:
The file was showing an error because the scss file wasn't successfully finding the image file - and adding this made it go away. If this does make it into the output CSS then it probably will cause an issue - I'll do some research!

Reply:
Ah yes, I've seen that error in the SCSS file.
I think Visual Studio / Rider is expecting the files to be in a particular folder (for the IntelliSense) but we've put them somewhere else (which work), but it confuses the IDE

Reply:
I think I agree with James here - lmk what the test results are but all else being equal I think without the `../..` is more likely to be correct, and you can stick in a linter rule to stop it complaining about this sort of thing